src/EditAnnotations.cpp

```diff
@@ -113,7 +113,7 @@ AnnotationType gAnnotsWithColor[] = {
 // clang-format on
 
 // in SumatraPDF.cpp
-extern void WindowInfoRerender(WindowInfo*);
+extern void WindowInfoRerender(WindowInfo*, bool = FALSE);

```

Comment:
When using bool, use true / false.
Also, name the bool argument. 
The rule is: if the argument is self-explanatory (like WindowInfo*), then we might skip naming it. Otherwise, it should be named.